docs/QuickStart-GettingStarted.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: getting-started
+id: quick-start-getting-started
 title: Getting Started
 layout: docs
 category: Quick Start
-permalink: docs/getting-started.html
-next: tutorial
+permalink: docs/quick-start-getting-started.html
```

Comment:
honestly, I don't think that we really care about the url itself. For the structure, I agree we need to make changes. Fortunately, you can just edit the category headers in markdown files and it'll show up in the right category